x-pack/plugins/cloud_security_posture/server/create_transforms/create_transforms.ts

```diff
@@ -36,7 +36,6 @@ export const createTransformIfNotExists = async (
   try {
     await esClient.transform.getTransform({
       transform_id: transform.transform_id,
-      allow_no_match: true,
```

Comment:
looks like that is the default behavior (https://www.elastic.co/guide/en/elasticsearch/reference/current/get-transform.html#get-transform-query-parms)